qiskit_ibm_experiment/service/dataclasses.py

```diff
@@ -0,0 +1,110 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Dataclasses for returned results"""
+from dataclasses import dataclass
+from typing import Optional, List, Dict, Any
+from datetime import datetime
+
+
+@dataclass
+class ExperimentData:
+    """Dataclass for experiments"""
+
+    experiment_id: str
```

Comment:
I think we should consider adding defaults to all fields to facilitate the use of the data classes here in qiskit-experiments. For example an `ExperimentData` object may be instantiated before we have an experiment id available.